README.md

```diff
@@ -65,14 +65,14 @@ Please refer to the [Agent Developer Guide](docs/dev/README.md) for more details
 
 ## Run
 
-To start the agent type `agent run` from the `bin/agent` folder, it will take
-care of adjusting paths and run the binary in foreground.
-
-You need to provide a valid API key. You can either use the config file or
-overwrite it with the environment variable like:
-```
-DD_API_KEY=12345678990 ./bin/agent/agent run -c bin/agent/dist/datadog.yaml
-```
+Running `invoke build` creates an Agent binary, located at `bin/agent/agent`.
+If the Datadog Agent is not installed, or if you do not want to edit your `datadog.yaml`, a configuration file is available at `bin/agent/dist/datadog.yaml`. To run the Agent with this configuration file, use `-c`: `DD_API_KEY=12345678990 bin/agent/agent <command> -c bin/agent/dist/` where `<command>` is an Agent command and `DD_API_KEY=12345678990` is a valid API key.
```

Comment:
for clarity, let's first have a code block with the simplest `run` command, so that it's immediately clear what a developer is supposed to run to run the built Agent.

Then (and only then IMO), this paragraph makes sense to explain how to run the dev Agent with a development `datadog.yaml` config file.